styleguide/source/assets/css/pattern-scaffolding.css

```diff
@@ -77,3 +77,7 @@
 .sg-subtype {
   padding: 5em 2em 0;
 }
+
+.sg-pattern-example .nav-primary-menu_overlay-mobile {
```

Comment:
@deadarm I think this should be `.nav-primary_overlay-mobile` instead of `.nav-primary-menu_overlay-mobile`. I'm still seeing the bogus overlay as-is.